catalog/USGS/NLCD_RELEASES/USGS_RCMAP_RELEASES_V5_COMPONENTS.jsonnet

```diff
@@ -0,0 +1,358 @@
+
+    Sagebrush is a continuous field component encompassing almost all species of Sagebrush (Artemisia spp.) including Big Sagebrush (A. tridentata spp.), Low Sagebrush (A. arbuscula), Black Sagebrush (A. nova), Three-tip Sagebrush (A. triparta) and Silver Sagebrush (A. cana). This component is nested within Shrub as a secondary component. Excludes the low stature prairie sage (A. frigida) and white sagebrush (A. ludoviciana).
+
+    Shrub is a continuous field component encompassing all shrub species discriminated by the presence of woody stems and &lt; 6-m in height.
+
+    Tree cover is defined as vegetation with persistent woody stems &gt; 6m in height. Mature stand of pinyon (Pinus spp.  and juniper (Juniperus spp.) are included regardless of height.
+
+    See also:
+    Hua Shi, Matthew Rigge, Kory Postma & Brett Bunde (2022) Trends analysis of rangeland condition monitoring assessment and projection (RCMAP) fractional component time series (1985–2020), GIScience & Remote Sensing, 59:1, 1243-1265, DOI: 10.1080/15481603.2022.2104786
+
+    Rigge, M., H. Shi, C. Homer, P. Danielson, and B. Granneman. 2019. Long-term trajectories of fractional component change in the Northern Great Basin, USA. Ecosphere: e02762.
+
+     Rigge, M., C. Homer, L. Cleeves, D. Meyer, B. Bunde, H. Shi, G. Xian, and M. Bobo. 2020. Quantifying western U.S. rangelands as fractional components with multi-resolution remote sensing and in situ data. Remote Sensing. 12: 412.
+
+     Rigge, M., Homer, C., Shi, H., Meyer, D., Bunde, B., Granneman, B., Postma, K., Danielson, P., Case, A., and Xian, G. 2021. Rangeland Fractional Components Across the Western United States from 1985 to 2018. Remote Sensing, 13: 813.
+|||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id) + [
+  ],
+  keywords: [
+    'shrublands',
+    'time series',
+    'back-in-time',
+    'trends',
+    'grassland change',
+    'shrubland change',
+    'vegetation change',
+    'climate change',
+    'rangeland management',
+    'shrub',
+    'sagebrush',
+    'big sagebrush',
+    'herbaceous',
+    'annual herbaceous',
+    'litter',
+    'grass',
+    'vegetation',
+    'bare ground',
+    'rangeland',
+    'shrubland',
+  ],
+  providers: [
+    ee.producer_provider('United States Geological Survey and Bureau of Land Management', 'https://www.mrlc.gov/'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(
+      -128.0026, 26.4827, -99.6407, 51.5777,
+      '1985-01-01T00:00:00Z',
+      '2021-12-31T00:00:00Z'),
+  'sci:citation': |||
+    Rigge, M.B., Bunde, B., Postma, K., Shi, H., 2022, Rangeland Condition Monitoring Assessment and Projection (RCMAP) Fractional Component Time-Series Across the Western U.S. 1985-2021: U.S. Geological Survey data release, https://doi.org/10.5066/P9ODAZHC.
+    [doi:10.5066/P9ODAZHC](https://doi.org/10.5066/P9ODAZHC)
+  |||,
+  'gee:user_uploaded': true,
```

Comment:
please also add
  'gee:skip_indexing': true,
  (this is a temporary thing our system needs right now)

Reply:
Added this line as line 108